adobe-students/scripts/utils.js

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/*
+ * ------------------------------------------------------------
+ * Edit below at your own risk
+ * ------------------------------------------------------------
+ */
+
+/**
+ * The decision engine for where to get Milo's libs from.
+ */
+export const [setLibs, getLibs] = (() => {
+  let libs;
+  return [
+    (prodLibs, force = false) => {
+      if (force) {
+        libs = prodLibs;
+        return libs;
+      }
+      const { hostname } = window.location;
+      if (
+        !hostname.includes('hlx.page')
+        && !hostname.includes('hlx.live')
+        && !hostname.includes('localhost')) {
+        libs = prodLibs;
+        return libs;
+      }
+      const branch = new URLSearchParams(window.location.search).get('milolibs') || 'main';
+      if (branch === 'local') { libs = 'http://localhost:6456/libs'; return libs; }
+      if (branch.indexOf('--') > -1) { libs = `https://${branch}.hlx.live/libs`; return libs; }
+      libs = `https://${branch}--milo--adobecom.hlx.live/libs`;
+      return libs;
+    }, () => libs,
+  ];
+})();
+
+const miloLibs = setLibs('/libs');
```

Comment:
we dont need the cc specific imports and functions. We can have this basic from milo-college